src/domains/User/UserSettings/UserSettings.tsx

```diff
@@ -143,6 +144,17 @@ export default function UserSettings({ id }: Props) {
             description: 'Customize notifications receieved',
             component: <NotificationSettings user={user} />,
         },
+        {
+            title: 'GeneralEditableText',
```

Comment:
Don't forget to add spaces

Reply:
Removed `GeneralEditableText` since it does not look as clean as the other `TextField` UserProfile. 

At first I had two profiles, one with `TextField` and the other with `EditableText` which I removed.

I can add it back and flush it out more if you'd like.